src/Components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { BookmarksIcon, ExploreIcon, HalfMoonIcon, HomeIcon, LogoutIcon, MultipleHeartIcon, ProfileIcon, SunIcon } from '../../Assets/AllSVG.jsx';
+import { useTheme } from '../../Context';
+
+import './Navbar.css';
+const Navbar = () => {
+	const [isSideBarOpen, setIsSideBarOpen] = useState(false);
+	const { theme, toggleTheme } = useTheme();
+	const openSideBar = () => setIsSideBarOpen(true);
+	const closeSideBar = () => setIsSideBarOpen(false);
+
+	return (
+		<>
+			<nav className='navbar'>
+				<NavLink className='navbar-brand ' to='/'>
+					Gamers Connect
+				</NavLink>
+				<input type='search' className='text-field' placeholder='Search' />
+				<div className='navbar-cta'>
+					<div
+						onClick={() => (!isSideBarOpen ? openSideBar() : closeSideBar())}
+						className={`menu-burger ${isSideBarOpen && 'menu-burger-opened'}`}
+					>
+						<div></div>
```

Comment:
If possible remove the empty div tags. one small suggestion.

Reply:
done.

Reply:
These div's make the hamburger icon can't remove them.